Week3/homework/step2-3.js

```diff
@@ -17,9 +17,9 @@ console.log('for', repeatStringNumTimesWithFor('abc', 3));
 function repeatStringNumTimesWithWhile(str, num) {
   // eslint-disable-next-line
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  while (result.length < str.length * num) {
```

Comment:
This loop condition looks overly complex. For every loop iteration a computation must be performed. Here, it would be more efficient and easier to understand if you just mimic the `for` loop, by introducing a loop index variable:

```js
let i = 0;
while (i < num) {
  //...
  i++;
}
```